app/Http/Controllers/Auth/AuthController.php

```diff
@@ -85,16 +86,15 @@ public function redirectToProvider(){
     public function handleProviderCallback(){
         try{
             $user = Socialite::driver('facebook')->user();
-        } catch( Exception $e){
-            return redirect('auth/facebook');
+        } catch (\Exception $e) {
```

Comment:
Please can you remove the backslash `\` and import Exception at the top. It makes the code look cleaner.